modules/loader/src/main/java/com/opengamma/strata/loader/csv/CsvLoaderUtils.java

```diff
@@ -723,4 +737,8 @@ public static String formattedDouble(double value) {
     return Decimal.of(value).toString();
   }
 
+  static ParseFailureException missingFieldException(String field) {
```

Comment:
Shouldn't this be in the resolver if it is going to be nicluded

Reply:
I included it here so we can use it in other parsers within and outside of Strata. Happy to keep it within the resolver if you think that is better.